index.js

```diff
@@ -1 +1,25 @@
-// JS code goes here
\ No newline at end of file
+// JS code goes here
+const hamburger = document.querySelector('.hamburger')
+const mobnav = document.querySelector('.mob__nav__menu')
+const nvsection = document.querySelector('.nv__section')
+const hamburger2 = document.querySelector('.mob__nav__menu .hamburger2')
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active')
+  mobnav.classList.toggle('active')
+  nvsection.classList.toggle('active')
+})
+
+document.querySelectorAll('.nv__item').forEach((n) =>
+  n.addEventListener('click', () => {
+    hamburger.classList.remove('active')
+    hamburger.style.display = 'block'
+    mobnav.classList.toggle('active')
+  })
+)
+
+hamburger2.addEventListener('click', () => {
+  hamburger.style.display = 'block'
+  hamburger.classList.remove('active')
+  mobnav.classList.toggle('active')
+})
```

Comment:
- [ ] Great job 👍 everything checks out well but I think you should try to use `camelCasing` convention in your variable name declaration when you are joining 2 or more words together. This is best practice and it allows more clarity when reading code. Examples of where you could use this are found in `mobnav` which could have been `mobNav` and `nvsection` as `nvSection` 😉 . This will increase the understandability of your code such you won't have issues remembering what is what.